set.mm

```diff
@@ -619955,6 +619979,73 @@ fixed reference functional determined by this vector (corresponding to
       ( clvec wcel clmod crg lveclmod lmodring syl ) BDEBFEAGEBHABCIJ $.
   $}
 
+  $( The property of modules to be vector spaces is invariant.  (Contributed by
+     Steven Nguyen, 15-Aug-2023.) $)
+  lmhmlvec $p |- ( F e. ( S LMHom T ) -> ( S e. LVec <-> T e. LVec ) ) $=
+    ( clmhm co wcel clmod csca cfv cdr wa lmhmlmod1 lmhmlmod2 2thd eqid lmhmsca
+    clvec eqcomd eleq1d islvec anbi12d 3bitr4g ) CABDEFZAGFZAHIZJFZKBGFZBHIZJFZ
+    KAQFBQFUCUDUGUFUIUCUDUGABCLABCMNUCUEUHJUCUHUEABCUEUHUEOZUHOZPRSUAUEAUJTUHBU
+    KTUB $.
+
+  ${
+    $d I x y $.  $d K u x y $.  $d F x y $.  $d W t u x y $.  $d I t u $.
+    frlmsnic.w $e |- W = ( K freeLMod { I } ) $.
+    frlmsnic.1 $e |- F = ( x e. ( Base ` W ) |-> ( x ` I ) ) $.
+    $( A module whose vectors map a single index to a value is isomorphic to
+       the module whose vectors are a value.  (Contributed by Steven Nguyen,
+       18-Aug-2023.) $)
```

Comment:
The result is more like:
Given a free module with a singleton as base, the function that maps each vector to its coordinate in that base is a module isomorphism from that module to its ring of scalars seen as a module.